.gitignore

```diff
@@ -1,2 +1,3 @@
 legal
 start
+3.0/
```

Comment:
Thanks for the fixes, Carter.
Were you just curious to see the software run,
or did you actually want to verify some of the results?

regards,
-John